### PR TITLE
jamf_protect: update manifest format version to v3.0.3

### LINKED ISSUE #9905: [jamf_protect] Mark sensitive values as secret
There are two configuration values flagged in #8610 that need to be marked as secret (or not a secret if it doesn't contain a sensitive value):

- http_endpoint.secret_header 
- http_endpoint.secret_value

In order to support secrets variables and validation, the package-spec version of the integration will need to be raised to 3.0.2 or higher. As a note, migrating to package-spec 3.0 or higher will more than likely trigger validation errors for fields.

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

See title.

Note removal of ECS fields. The probably best approach to address these issues would be to retain a jamf_protect object to hold items like these. This is not done in this change.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- Closes #9905
- For #8610

### CODE REVIEW

File: packages/jamf_protect/data_stream/alerts/fields/ecs.yml [hunk 1]
```diff
@@ -1,267 +1,191 @@
...
-- external: ecs
-  name: event.category
-- external: ecs
-  name: event.id
-- external: ecs
-  name: event.timezone
+  name: container.image.tag
 - external: ecs
-  name: related.ip
+  name: container.runtime
```
Comment (kcreddy) [inline packages/jamf_protect/data_stream/alerts/fields/ecs.yml:4]: What are the changes in this file?

File: packages/jamf_protect/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -393,7 +392,7 @@ processors:
         if: ctx.jamf_protect?.alerts?.input?.match?.facts[0].tags.contains("MITREattack")
     - set:
         field: threat.software.platforms
-        value: macOS
+        value: [macOS]
```
Comment (kcreddy) [inline packages/jamf_protect/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml:395]: Would it be better if we used `append` here?

**Review by kcreddy — APPROVED** [review]
nits
LGTM 👍🏼

File: packages/jamf_protect/data_stream/alerts/fields/ecs.yml (hunk 1 above)
Comment (efd6) [reply packages/jamf_protect/data_stream/alerts/fields/ecs.yml:4]: That is correct.

File: packages/jamf_protect/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml (hunk 2 above)
Comment (efd6) [reply packages/jamf_protect/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml:395]: This is the only time this field is being reference. Would you prefer that I use `append`?

Comment (kcreddy) [reply packages/jamf_protect/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml:395]: Thats fine 👍🏼

**Pull request merged**